include/swift/AST/DiagnosticsSIL.def

```diff
@@ -491,6 +491,10 @@ WARNING(autodiff_nonvaried_result_fixit,none,
         "result does not depend on differentiation arguments and will always "
         "have a zero derivative; do you want to use 'withoutDerivative(at:)'?",
         ())
+// TODO(TF-1149): Remove this diagnostic.
+NOTE(autodiff_loadable_value_addressonly_tangent_unsupported,none,
+     "cannot differentiate value with loadable type %0 but address-only "
```

Comment:
Diagnostics should be user-facing instead of compiler-engineer-facing. How about:

> cannot yet differentiate value with a compile-time known size but a `TangentVector` that contains stored properties of unknown size; consider modifying the `TangentVector` to use fewer generic parameters in stored properties

Reply:
Thanks for the tip! Adapted in https://github.com/apple/swift/pull/29749/commits/610214a9c82960e66007edbbf78336a30f310e90:

```
cannot yet differentiate value whose type 'Class<T>' has a compile-time known size, but whose 'TangentVector' contains stored properties of unknown size; consider modifying 'Class<T>.TangentVector' to use fewer generic parameters in stored properties
```